### ISSUE TITLE
Unable to add The Horror Charnel indexer

### ISSUE THREAD

**Issue by littleblue50** [Prowlarr/Prowlarr#1065]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

I try to add The Horror Charnel indexer and test/save and it fails with the following error: Unable to connect to indexer. Unexpected response status Redirect code from API request

### Expected Behavior

The indexer should save successfully and add to the indexer list

### Steps To Reproduce

Try to add The Horror Charnel as an indexer with valid credentials.

### Environment

```markdown
- OS: Synology NAS
- Prowlarr: 0.4.2.1879
- Docker Install: Yes
- Using Reverse Proxy: No
- Browser: N/A (All)
```

### What branch are you running?

Develop

### Trace Logs?

https://pastebin.com/uQzLuXh6

**Comment by bakerboy448** [comment]
Prior Issue - https://github.com/Prowlarr/Prowlarr/issues/122

appears the cookie is not being used? Or not gotten from the login

**Comment by bakerboy448** [comment]
Duplicate of  https://github.com/Prowlarr/Prowlarr/issues/122

hopefully was fixed with 0.4.3.1881
https://github.com/Prowlarr/Prowlarr/commit/c53e0054ee3f075482f0457464924b9bbea7987d
